src/OperationalInsights/OperationalInsights/Workspaces/NewAzureOperationalInsightsWorkspaceCommand.cs

```diff
@@ -42,7 +42,7 @@ public class NewAzureOperationalInsightsWorkspaceCommand : OperationalInsightsBa
 
         [Parameter(Position = 3, Mandatory = false, ValueFromPipelineByPropertyName = true,
             HelpMessage = "The service tier of the workspace.")]
-        [ValidateSet("free", "standard", "premium", "pernode","standalone", IgnoreCase = true)]
+        [ValidateSet("free", "standard", "premium", "pernode", "standalone", "pergb2018", IgnoreCase = true)]
```

Comment:
Please change this to a PSArgumentCompleter - this allows the user to tab through known values, but enter a freeform value as well, so that updating the cmdlet when a new sku is available is less urgent

Reply:
Here is an example usage: https://github.com/Azure/azure-powershell/blob/32366997ce9d2e7d38fd44ac6d8a24353ffea64f/src/Cdn/Cdn/Endpoint/NewAzCdnDeliveryRuleAction.cs#L45

Reply:
not sure I understand, what does it mean a freeform value ? 
does this attribute also validates that the value is from the given set ?

Reply:
It does not - it allows the user to tab through the available values, and enter a value outside the set.  Users can enter a value outside the set, but are directed to known values, which is why this is now the recommended pattern, rather than validateSets

Reply:
okay, sure sounds like a better approach. I will remove validateset